system-addon/lib/PingCentre.jsm

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const {interfaces: Ci, utils: Cu} = Components;
+Cu.import("resource://gre/modules/Preferences.jsm");
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+Cu.importGlobalProperties(["fetch"]);
+
+XPCOMUtils.defineLazyModuleGetter(this, "ClientID",
+  "resource://gre/modules/ClientID.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "console",
+  "resource://gre/modules/Console.jsm");
+
+const PREF_BRANCH = "browser.ping-centre.telemetry.";
+
+const TELEMETRY_PREF = `${PREF_BRANCH}telemetry`;
```

Comment:
I didn't do this because PingCentre also references `FHR_UPLOAD_ENABLED_PREF` from its `this._prefs` object so it would need to access its different prefs in different ways which seemed more verbose to me. Let me know if you disagree.